internal/internal_task_handlers.go

```diff
@@ -1273,7 +1287,7 @@ func lastPartOfName(name string) string {
 	return name[lastDotIdx+1:]
 }
 
-func isCommandMatchEvent(d *commandpb.Command, e *historypb.HistoryEvent, strictMode bool) bool {
+func isCommandMatchEvent(d *commandpb.Command, e *historypb.HistoryEvent) bool {
```

Comment:
This `strictMode ` flag was never used and wouldn't work if the go version was changed